cdap-explore/src/main/java/co/cask/cdap/hive/context/ContextManager.java

```diff
@@ -78,11 +100,12 @@ public static Context getContext(@Nullable Configuration conf) throws IOExceptio
       // by the DatasetSerDe.initialize method, which needs to access the context when it wants to write to a
       // dataset, so it can cache its name. In that case, conf will be null, and it won't be possible to create a
       // context.
-      savedContext = createContext(conf);
+      return createContext(conf);
```

Comment:
there's comment above this line about why saving is important.. is that explanation not valid ? should we remove that then?